packages/matchbox/package.json

```diff
@@ -61,7 +64,10 @@
     "rollup-plugin-commonjs": "^10.1.0",
     "rollup-plugin-node-resolve": "^5.2.0",
     "rollup-plugin-terser": "^7.0.2",
-    "styled-components": "^5.2.1"
+    "rollup-plugin-typescript2": "^0.30.0",
+    "styled-components": "^5.2.1",
+    "tslib": "^2.3.0",
+    "typescript": "^4.3.5"
```

Comment:
Will also need to export definitions from this file `"types": "./types.d.ts"`

Reply:
Good point